core/src/main/java/com/datastax/oss/driver/internal/core/ssl/SniSslEngineFactory.java

```diff
@@ -38,9 +38,15 @@ public class SniSslEngineFactory implements SslEngineFactory {
 
   private final SSLContext sslContext;
   private final CopyOnWriteArrayList<String> fakePorts = new CopyOnWriteArrayList<>();
+  private final boolean allowDnsReverseLookupSan;
 
   public SniSslEngineFactory(SSLContext sslContext) {
+    this(sslContext, true);
+  }
+
+  public SniSslEngineFactory(SSLContext sslContext, boolean allowDnsReverseLookupSan) {
```

Comment:
Didn't dawn on me at the time that `SniSslEngineFactory` does not access `DriverContext` so we can't make use of it unless programatically (outside of `CloudConfigFactory`), but I suppose if someone wants to use it separately, they can programmatically and in which case this adds some functionality :+1: